packages/browser/src/index.ts

```diff
@@ -240,4 +276,22 @@ export default class LogtoClient {
       throw new LogtoClientError('invalid_id_token', error);
     }
   }
+
+  private saveCodeToken({
+    refreshToken,
+    idToken,
+    scope,
+    accessToken,
+    expiresIn,
+  }: CodeTokenResponse) {
+    localStorage.setItem(`${this.logtoStorageKey}:${refreshTokenKey}`, refreshToken);
+    localStorage.setItem(`${this.logtoStorageKey}:${idTokenKey}`, idToken);
+
+    this.refreshToken = refreshToken;
+    this.idToken = idToken;
+
+    const accessTokenKey = buildAccessTokenKey('', scope.split(' '));
```

Comment:
Since we don't use scope to get access token in v1, I'm afraid the scope should not be used here, either.

Reply:
Updated.

Added Linear issue LOG-1589.
(See [discussion](https://silverhand-io.slack.com/archives/C02A8G4HVAM/p1645156874710139) on Slack)